### DISCUSSION TITLE
안녕하세요 튜토리얼 중 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by ce-dric** [reinforcement-learning-kr/2021_RLKR_Drone_Delivery_Challenge_with_Unity#14] [General]
이번 기회에 RL 들여다보려구 하는데..
[기본 Command 예시](https://github.com/reinforcement-learning-kr/2021_RLKR_Drone_Delivery_Challenge_with_Unity/blob/master/docs/run_with_ml-agents.md) 에서

```md
(Windows build 파일 기준)
YourDirectory
├── RLVillage
│   ├── DroneDelivery.exe
│   ├── DroneDelivery_Data  
│   ├── MonoBleedingEdge  
│   ├── UnityCrashHandle64.exe             
│   └── UnityPlayer.dll
└── drone_config
    └── ppo.yaml
```

제공해주신 파일에는 `ppo.yaml` 파일이 없는데, 원래 제공되지 않는건가요?

**Comment by curieuxjy** [comment]
안녕하세요!

네 해당 `ppo.yaml`파일은 제공되는 것이 아니고 질문에 달아주신 파일 구조에서 `RLVillage`가 제공된 파일입니다.(Windows 기준)

`ppo.yaml`파일은 mlagents 내부 알고리즘으로 학습할 경우에 작성해야하는 config 파일로,
[Behavior Config](https://github.com/reinforcement-learning-kr/2021_RLKR_Drone_Delivery_Challenge_with_Unity/blob/master/docs/run_with_ml-agents.md#behavior-config)에서 설명하고 있는 config 파일이 `.yaml` 확장자로 저장되는 파일이라고 보시면 됩니다.

yaml 파일을 작성하실 때, 기본 포맷은 [Unity 공식 문서](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Training-ML-Agents.md#behavior-configurations)를 참고하셔서 작성하신 후 `.yaml`파일로 생성하여 실행하시면 됩니다.

**Reply by ce-dric** [reply]
답변감사합니다 ㅎㅎ

**Comment by pnltoen** [comment]
RL KOREA 운영진은 아니지만 도움이 되실까해서 답변드립니다!
Hyperparameter 튜닝을 위한 .yaml 확장 파일은 unity-mlagents 공식 깃허브에서도 다운로드가 가능합니다.
https://github.com/Unity-Technologies/ml-agents 들어가셔서 코드 다운로드 또는 git clone https://github.com/Unity-Technologies/ml-agents 하신 후 config/ppo 폴더에 보시면 mlagents 예제에 맞는 각각의 .yaml 파일을 찾으실 수가 있습니다!

**Reply by ce-dric** [reply]
답변 감사합니다!
